.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+#### Link to ticket
```

Comment:
This is is a part of https://github.com/eReolen/ereolen/pull/39 and should not be included in this PR.